EnglishFilter/sections/css_extended.txt

```diff
@@ -610,6 +610,7 @@ rarbgproxied.org,rarbg.to,rarbg-to.proxydude.xyz,rarbg-to.pbproxy.red,rarbg.unbl
 alluc.ee#?##resultlist > div#resultitems[-ext-has="> div"]~iframe:not([src])
 !
 spankwire.com#?##js-react-video-player > div > div > div:has(> a[href="https://www.czechsexcasting.com/"])
+designoptimal.com#?#.widget-area > aside.sidebar-widget:has(> div.textwidget ins.adsbygoogle)
```

Comment:
This rule hides Google search.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/65766929-aab04480-e12c-11e9-83e2-c9093c1f4d3b.png)

</details>

Maybe:
`designoptimal.com#?#.widget-area > aside.sidebar-widget:has(> div.textwidget:only-child ins.adsbygoogle)`